src/i18n/en.json

```diff
@@ -1,180 +1,191 @@
 {
-  "title": "English",
   "app_title": "ADAMANT Messenger",
-  "new_messages_1": "new message",
-  "new_messages_2": "new messages",
-  "new_messages_5": "new messages",
-  "region": "en-US",
-  "error": "Error",
-  "login": {
-    "copy_button_tooltip": "Click to copy the passphrase to clipboard. Save it in a password keeper application. To enter your new Wallet, paste it the passphrase field.",
-    "save_button_tooltip": "Click to save your passphrase in a text file. Save it in the secure place!",
-    "password_label": "Input your passphrase to login",
-    "new_password_label": "<b>Save</b> the passphrase for new Wallet and Messenger account. There is no login to enter Wallet, only the passphrase needed. If lost, no way to recover it.",
-    "language_label": "Select your language",
-    "login_button": "Login",
-    "new_button": "Create new",
-    "invalid_passphrase": "Invalid passphrase",
-    "invalid_qr_code": "This QR code does not include valid passphrase",
-    "device_unsupported": "Your device is not supported",
-    "subheader": "A top-notch anonymous messenger encrypted with a Blockchain",
-    "create_address_label": "Or create a new anonymous address:",
-    "scan_qr_code_button_tooltip": "Scan QR with passphrase via device Camera"
-  },
-  "options": {
-    "version": "App Version: ",
-    "language_label": "Language: ",
-    "exit_on_close": "Logout on tab close",
-    "exit_on_close_tooltip": "For your wallet security we recommend you keep this checked. In case uou'll uncheck it, your passphrase will be saved in browser storage, and next time you open the wallet it will be decripted with no need to enter a passphrase. It is acceptable only in case noone else can access your device.",
-    "send_on_enter": "Send message on Enter",
-    "send_on_enter_tooltip": "Send message on Enter key, new line on Shift + Enter. By default combination Ctrl (Cmd) + Enter is used to send messages.",
-    "general_title": "General",
-    "security_title": "Security",
-    "chats_title": "Chats",
-    "notification_title": "Notifications",
-    "enable_sound": "Sound",
-    "enable_sound_tooltip": "Play a sound when new message arrives. It works not on all of devices. Mobile browsers will not play a sound when browser is not active.",
-    "enable_bar": "Change browser tab title",
-    "enable_bar_tooltip": "Shows new message counter in the tab title when new message arrives. Works for desktop browsers.",
-    "enable_desktop": "Browser notifications",
-    "enable_desktop_tooltip": "Use integrated browser notifications (need your acceptance)",
-    "delegates_title": "Delegates",
-    "delegate_name_input": "Enter delegate name",
-    "delegate_register_button": "Register delegate",
-    "vote_for_delegates_button": "Vote for delegates",
-    "already_delegate_text": "You are already a delegate. Your delegate name is"
+  "bottom": {
```

Comment:
If we resorted language constants in english, shouldn't we do the same for other languages?